clients/extension/src/content/index.ts

```
@@ -620,22 +629,75 @@ namespace Provider {
+          } catch {
+            console.warn(
+              'Receiver token account not found. Checking for ATA...'
+            )
+            const ataInstruction = transaction.instructions.find(instr =>
+              instr.programId.equals(ASSOCIATED_TOKEN_PROGRAM_ID)
+            )
+            if (ataInstruction) {
+              // The recipient should be in the ATA instruction's keys[0] (payer) or keys[2] (owner)
+              recipient = ataInstruction.keys[2].pubkey.toString()
+            } else {
+              throw new Error('Unable to determine recipient address. No direct token account or ATA instruction found.')
```

ESLint

[error] Replace `'Unable·to·determine·recipient·address.·No·direct·token·account·or·ATA·instruction·found.'` with `⏎················'Unable·to·determine·recipient·address.·No·direct·token·account·or·ATA·instruction·found.'⏎··············`

(prettier/prettier)